public/app/features/alerting/unified/mocks.ts

```diff
@@ -599,6 +599,10 @@ export const grantUserPermissions = (permissions: AccessControlAction[]) => {
     .mockImplementation((action) => permissions.includes(action as AccessControlAction));
 };
 
+export const grantUserRole = (role: string) => {
```

Comment:
I feel we should try to use `grantUserPermission` instead of roles. Roles are only abstraction over permissions and might give incorrect results when RBAC is enabled

Reply:
Completely agree – though the admin section is an exception here and we do check for the role if I recall correctly?

Reply:
Ah, you're probably right. There is no special permission that would give access to this page